README.md

```diff
@@ -23,7 +23,8 @@ npm install bikram-samvat-calendar
  ```bash 
  import React, { useState } from 'react';
 import Calendar from 'bikram-samvat-calendar';
-import { NepaliDate } from "bikram-samvat-calendar/build/types/Calender";
+import { NepaliDate } from 'bikram-samvat-calendar/types/Calender';
```

Comment:
spelling should be `calendar` not `calender`. lets fix that in a single pass separately @kapil-31